azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -320,6 +323,25 @@ public void run() {
     }
   }
 
+  @Override
+  public void cancelFlow(ExecutableFlow exFlow, String userId)
```

Comment:
What's the difference between method in AbstractExecutorManagerAdapter and this?

Reply:
The implementation in `AbstractExecutorManagerAdapter` explicitly relies on the existence of a non-null `executor` in the ExecutionReference which result in the failure of cancelFlow. For ContainerizedDispatch a valid executor is not required and will typically not be present. (If it does exist for any reason, ApiGateway calls have already been modified to handle it correctly)